packages/router-plugin/package.json

```diff
@@ -6,8 +6,8 @@
   "sideEffects": true,
   "peerDependencies": {
     "@ngxs/store": "^0.0.0",
-    "@angular/core": ">=5.0.0 <8.0.0",
-    "@angular/router": ">=5.0.0 <8.0.0",
+    "@angular/core": ">=5.0.0 <9.0.0",
+    "@angular/router": ">=5.0.0 <9.0.0",
```

Comment:
As per this twit https://twitter.com/stephenfluin/status/1129429898047643648 suggested to set peer dependency to ^8.0.0 || ^9.0.0-0.

Reply:
Good catch